<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GerirModulo</class>
 <widget class="QDialog" name="GerirModulo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gerir M贸dulo</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../icon.ico</normaloff>../icon.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background:white
</string>
  </property>
  <widget class="QTableView" name="tableViewTipodeMensagemModulo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>761</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="commandButtonVoltar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-10</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../setavoltar.ico</normaloff>../setavoltar.ico</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="btnApagarModulo">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>140</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:rgb(255, 245, 166);
	border:none;
	color:rgb(95, 91, 61);
	border -bottom: 5px solid rgb(162, 155, 105)
}
QPushButton:hover{
	background-color:rgb(229, 217, 150);
	border-bottom: 5px solid rgb(162, 155, 105)
}
QPushButton:pressed {
	background-color:rgb(255, 242, 165);
	border-bottom: 5px solid rgb(162, 155, 105)
}</string>
   </property>
   <property name="text">
    <string>Apagar M贸dulo</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>190</y>
     <width>791</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>M贸dulo</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableViewModulo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>761</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnGuardarModulo">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>140</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:rgb(255, 245, 166);
	border:none;
	color:rgb(95, 91, 61);
	border -bottom: 5px solid rgb(162, 155, 105)
}
QPushButton:hover{
	background-color:rgb(229, 217, 150);
	border-bottom: 5px solid rgb(162, 155, 105)
}
QPushButton:pressed {
	background-color:rgb(255, 242, 165);
	border-bottom: 5px solid rgb(162, 155, 105)
}</string>
   </property>
   <property name="text">
    <string>Guardar</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelNomeModulo">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>modulo xpto</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>441</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Tipo de Mensagem deste M贸dulo:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCriarTipoMensagem">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>450</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:rgb(255, 245, 166);
	border:none;
	color:rgb(95, 91, 61);
	border -bottom: 5px solid rgb(162, 155, 105)
}
QPushButton:hover{
	background-color:rgb(229, 217, 150);
	border-bottom: 5px solid rgb(162, 155, 105)
}
QPushButton:pressed {
	background-color:rgb(255, 242, 165);
	border-bottom: 5px solid rgb(162, 155, 105)
}</string>
   </property>
   <property name="text">
    <string>Criar Tipo de Mensagem</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
